apps/ensadmin/src/components/datetime-utils/index.tsx

```diff
@@ -1,24 +1,51 @@
-import { formatDistance, formatDistanceStrict, intlFormat } from "date-fns";
+import {
+  IntlFormatFormatOptions,
+  formatDistance,
+  formatDistanceStrict,
+  intlFormat,
+} from "date-fns";
 import { millisecondsInSecond } from "date-fns/constants";
 import { useEffect, useState } from "react";
 
 /**
- * Client-only date formatter component
+ * Format date as a datetime string.
  */
-export function FormattedDate({
-  date,
-  options,
-}: {
-  date: Date;
-  options: Intl.DateTimeFormatOptions;
-}) {
-  const [formattedDate, setFormattedDate] = useState<string>("");
+export function datetimeFormat(date: Date, args: Partial<IntlFormatFormatOptions> = {}): string {
```

Comment:
```suggestion
export function formatDatetime(date: Date, args: Partial<IntlFormatFormatOptions> = {}): string {
```

Goal: For functions that perform some string formatting, it's best to make "format" (the verb) the prefix, not the suffix.